src/main/groovy/com/blackducksoftware/integration/hub/detect/hub/PolicyChecker.groovy

```diff
@@ -66,4 +69,34 @@ class PolicyChecker {
         logger.info("Policy Status: ${statusEnum.name()}")
         policyStatusDescription
     }
+
+    public boolean policyViolated(PolicyStatusDescription policyStatusDescription) {
```

Comment:
Should this method also be checking severity? or should it just be checking if a policy was violated?

Reply:
Ok, it seems like we always want to check if policies of a configured list of severities have been violated and the default is all severities.  Would the method name arePolicySeveritiesViolated() make more sense?